components/resources/ResourceSearch.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Input from "../../components/Input";
+import styles from "../../styles/modules/Resources.module.scss";
```

Comment:
Is it standard to use parent path locations for imports that traverse multiple levels?  ../../. ?  Not a big deal I assume, just wanted to know if there's a loader function that helps keep all imports at the same level.  Just asking.

Reply:
There are ways to set up the project to treat specific directories like top level imports. In my experience, I haven't seen that done much, but I like the idea because the imports are a little ugly. I'll do some research to see if there's any downsides to doing that for some of the frequently imported directories.

Reply:
The good thing is here they're easy enough to find.  I think it would get interesting at 3+ levels deep though.  This could be faster too... something to look at when there's not much else maybe.

Reply:
Good thinking, it was super easy to implement for both the test and the implementation files so it will be clean(er) imports from here on out 🏄